### PR TITLE
Reset review vote option

### PR DESCRIPTION
**Pull request by Durafen**
Closes [No way to remove a review vote #195](https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/195)

### CODE REVIEW

File: www/reviews.php [hunk 1]
```diff
@@ -439,6 +445,8 @@ function showReview($db, $gameid, $rec, $specialNames, $optionFlags = 0)
             $result = mysql_query(
                 "select vote from reviewvotes
                  where reviewid = '$qreviewid' and userid = '$curuser'", $db);
+                 error_log("select vote from reviewvotes
+                 where reviewid = '$qreviewid' and userid = '$curuser'");
```
Comment (dfabulich) [inline www/reviews.php:449]:
```suggestion
```

File: www/reviewvote [hunk 2]
```diff
@@ -75,23 +75,33 @@ mysql_query("delete from reviewvotes
...
+	} else {
+	    sendResponse("Not Saved", "An error occurred updating the database. "
+	                 . "Please try again later.", false);
+	}
 } else {
-    sendResponse("Not Saved", "An error occurred updating the database. "
-                 . "Please try again later.", false);
+// Reset Vote result	
+    sendResponse("Vote Reseted - Thanks!", false,
+                 "Thanks for voting! Your vote has been reseted.", false);
```
Comment (dfabulich) [inline www/reviewvote:100]:
```suggestion
    sendResponse("Vote Removed - Thanks!", false,
                 "Thanks for voting! Your vote has been removed.", false);
```

"Reseted" isn't an English word (though it totally ought to be)

File: www/reviews.php [hunk 3]
```diff
@@ -450,7 +458,10 @@ function showReview($db, $gameid, $rec, $specialNames, $optionFlags = 0)
             . "return false;\">Yes</a> &nbsp; "
             . "<a href=\"needjs\""
             . "onclick=\"javascript:sendReviewVote('$reviewid', 'N');"
-            . "return false;\">No</a> &nbsp; ";
+            . "return false;\">No</a> &nbsp; "
+            . "<a href=\"needjs\""
+            . "onclick=\"javascript:sendReviewVote('$reviewid', 'R');"
+            . "return false;\">Reset</a> &nbsp; ";
```
Comment (dfabulich) [inline www/reviews.php:464]:
```suggestion
            . "return false;\">Remove vote</a> &nbsp; ";
```

Seems like on https://github.com/iftechfoundation/ifdb-suggestion-tracker/issues/195 folks liked "Remove vote"

**Review by dfabulich — CHANGES_REQUESTED** [review]
Nice work! A few suggestions

File: www/reviews.php [hunk 4]
```diff
@@ -450,7 +456,10 @@ function showReview($db, $gameid, $rec, $specialNames, $optionFlags = 0)
             . "return false;\">Yes</a> &nbsp; "
             . "<a href=\"needjs\""
             . "onclick=\"javascript:sendReviewVote('$reviewid', 'N');"
-            . "return false;\">No</a> &nbsp; ";
+            . "return false;\">No</a> &nbsp; "
+            . "<a href=\"needjs\""
+            . "onclick=\"javascript:sendReviewVote('$reviewid', 'R');"
+            . "return false;\">Remove vote</a> &nbsp; ";
```
Comment (dfabulich) [inline www/reviews.php:462]: I just noticed that it _always_ includes the "Remove vote" link, even if you've never voted, and even right after you've removed your vote. (But fixing that sounds like a hassle.)

WDYT? Do you want to take a crack at making the link only show up when it's needed? Or I can just merge this now and we can file that as a separate bug?

**Comment by Durafen** [comment]
Updated it, so it dynamically updates the "Remove vote" option.

**Comment by dfabulich** [comment]
Thanks! (Sorry for the delay in reviewing.)

**Pull request merged**